test/e2e/performanceprofile/functests/1_performance/cpu_management.go

```diff
@@ -329,35 +306,41 @@ var _ = Describe("[rfe_id:27363][performance] CPU Management", Ordered, func() {
 			Expect(err).ToNot(HaveOccurred())
 
 			By("Getting the CPU scheduling flags")
-			flags, err := getCPUsSchedulingDomainFlags()
+			// After the testpod is started get the schedstat and check for cpus
+			// not participating in scheduling domains
+			cpusNotInSchedulingDomains, err := getCPUsSchedulingDomains()
 			Expect(err).ToNot(HaveOccurred())
 
 			By("Verifying that the CPU load balancing was disabled")
 			for _, cpu := range cpus.ToSlice() {
-				Expect(len(flags[cpu])).To(Equal(len(defaultFlags[cpu])))
-				// the CPU flags should be almost the same except the LSB that should be disabled
-				// see https://github.com/torvalds/linux/blob/0fe5f9ca223573167c4c4156903d751d2c8e160e/include/linux/sched/topology.h#L14
-				// for more information regarding the sched domain flags
-				for i := range flags[cpu] {
-					Expect(flags[cpu][i]).To(Equal(defaultFlags[cpu][i] - 1))
+				for _, str := range cpusNotInSchedulingDomains {
+					if strings.Contains(str, fmt.Sprint(cpu)) {
+						cpuFound = true
+						break
+					}
 				}
 			}
+			Expect(cpuFound).To(BeTrue())
 
 			By("Deleting the pod")
 			deleteTestPod(testpod)
 
 			By("Getting the CPU scheduling flags")
-			flags, err = getCPUsSchedulingDomainFlags()
+			// Once the pod is deleted the cpus used by the testpod
+			// should not be found in scheduling
+			cpusNotinSchedulingDomains, err := getCPUsSchedulingDomains()
```

Comment:
The name of the function getCPUsSchedulingDomains should probably be changed.

Reply:
i have added some comments to conditions and also changed the function names